src/test/java/org/kainos/ea/team2/integration/AuthIntegrationTest.java

```diff
@@ -0,0 +1,58 @@
+package org.kainos.ea.team2.integration;
+
+import org.kainos.ea.team2.MovesLikeSwaggerApplication;
+import org.kainos.ea.team2.MovesLikeSwaggerConfiguration;
+import org.kainos.ea.team2.cli.BasicCredentials;
+
+import io.dropwizard.configuration.ResourceConfigurationSourceProvider;
+import io.dropwizard.testing.junit5.DropwizardAppExtension;
+import io.dropwizard.testing.junit5.DropwizardExtensionsSupport;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import javax.ws.rs.client.Entity;
+import javax.ws.rs.core.Response;
+
+
+@ExtendWith(DropwizardExtensionsSupport.class)
+public class AuthIntegrationTest {
+    static final DropwizardAppExtension<MovesLikeSwaggerConfiguration> APP = new DropwizardAppExtension<>(
+            MovesLikeSwaggerApplication.class, null,
+            new ResourceConfigurationSourceProvider()
+    );
+
+    /**
+     * Verify that logging in valid credentials gives back a valid JWT
+     */
+    @Test
+    void login_shouldReturnValidJWT_whenValidCredentialsArePassed() {
+        BasicCredentials credentials = new BasicCredentials("test@kainos.com","testing");
```

Comment:
you probably don't want to hardcode valid credentials, could you pull these in from env vars?

Reply:
Changed to pull from env vars.